rust-mozjs/src/jsglue.cpp

```diff
@@ -1018,6 +1018,14 @@ void EncodeStringToUTF8(JSContext* cx, JS::HandleString str,
   cb(chars.get());
 }
 
+JSString* JS_ForgetStringLinearness(JSLinearString* str) {
+  return JS_FORGET_STRING_LINEARNESS(str);
+}
+
+JSString* EnsureLinearString(JSContext* cx, JSString* str) {
+  return JS_FORGET_STRING_LINEARNESS(JS_EnsureLinearString(cx, str));
+}
+
```

Comment:
Can this API return a JSLinearString so that it works the same as the wrapped version?

Reply:
In that case, this API isn't needed at all. `JS_EnsureLinearString` can already be called from rust.